docs/checkers/index.md

```diff
@@ -1740,6 +1740,26 @@ class CashRegister:
 
 ## Exceptions
 
+(W0719)=
```

Comment:
Please move this to below W0718, to make the numbers align better. (Also double-check this for W0133 in your other PR)